Masonry/MASCompositeConstraint.m

```diff
@@ -18,9 +18,9 @@ @interface MASCompositeConstraint () <MASConstraintDelegate>
 
 @implementation MASCompositeConstraint
 
-- (id)initWithChildren:(NSArray *)children {
+- (instancetype)initWithChildren:(NSArray *)children {
     self = [super init];
-    if (!self) return nil;
+    if (!self) return self;
```

Comment:
this is a bit awkward, i know, but otherwise the compiler will complain that the method should not return `nil` due to nullability.